### PR TITLE
Added wrapper for fetching mnist

### PR DESCRIPTION
**Pull request by kmyim**

### CODE REVIEW

File: skdim/datasets.py [hunk 1]
```diff
@@ -60,6 +61,7 @@ def product(n, sample_1, sample_1_kwargs, sample_2, sample_2_kwargs):
     return np.hstack((sample_1_points, sample_2_points))
 
 
+
```
Comment (johnharveymath) [inline skdim/datasets.py:64]: Remove extra line

File: skdim/datasets.py [hunk 2]
```diff
@@ -576,6 +578,26 @@ def generate(self, n_points, depth=2):
         data = np.apply_along_axis(self._point_from_parameter, 1, parameters)
         return data
 
+def mnist(imshape=False, digit_class=True):
+    """Fetch MNIST dataset of 60000 28x28 greyscale handwritten digits from OpenML.
```
Comment (johnharveymath) [inline skdim/datasets.py:582]: Seems to be 70000

**Review by johnharveymath — CHANGES_REQUESTED** [review]
Looks great, thanks!

**Comment by kmyim** [comment]
Thanks, rectified the error in dimension

**Pull request merged**